lib/src/functions.dart

```diff
@@ -44,7 +44,7 @@ class TestFileInfo {
     }
     relative = relative.reversed.toList();
     final alias = relative.join('_').replaceFirst('.dart', '');
-    final importPath = relative.join(_sep);
+    final importPath = relative.join('/');
```

Comment:
Dart expects `import` statments to use `/` regardless of the platform its running on